src/rules/no-empty-file/__tests__/index.js

```diff
@@ -0,0 +1,61 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  skipBasicChecks: true,
+
+  accept: [ {
+    code: ".class { }",
+  }, {
+    code: "   .class { }   ",
+  }, {
+    code: "/* comment */",
+  }, {
+    code: "\n.class {}",
+  }, {
+    code: "\r\n.class {}",
+  } ],
+
+  reject: [ {
+    code: "",
```

Comment:
Let's ensure that all these rejections occur at line 1 column 1, because otherwise I'd expect this to break text editor plugins.